packages/handlers/odata/src/schema-factory.ts

```diff
@@ -0,0 +1,537 @@
+import { GraphQLOutputType, GraphQLInputType, GraphQLString, GraphQLInt, GraphQLFloat, GraphQLBoolean, GraphQLScalarType, GraphQLEnumType, GraphQLEnumValueConfigMap, Kind, ObjectTypeDefinitionNode, GraphQLFieldConfigMap, GraphQLObjectType, GraphQLList, GraphQLInterfaceType, GraphQLNonNull, GraphQLInputObjectType, GraphQLInputFieldConfigMap, GraphQLID, GraphQLFieldConfigArgumentMap, GraphQLNamedType, GraphQLUnionType, GraphQLResolveInfo, GraphQLType } from "graphql";
+import { BigIntResolver as GraphQLBigInt, GUIDResolver as GraphQLGUID, DateTimeResolver as GraphQLDateTime } from 'graphql-scalars';
+import { KeyValueCache, Headers, Request, fetchache } from "fetchache";
+import urljoin from 'url-join';
+import { camelCase } from 'camel-case';
+import Interpolator from 'string-interpolation/src';
+import { JSDOM } from 'jsdom';
+import graphqlFields from 'graphql-fields';
+
+const SCALARS: [string, GraphQLScalarType][] = [
+    ['Edm.Binary', GraphQLString],
```

Comment:
According to the docs (https://www.odata.org/documentation/odata-version-2-0/overview/, section 6), the following scalars are missing: 
- Edm.Byte
- Edm.Decimal
- Edm.SByte

Reply:
Added!